android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/notifications/OnSystemRequestTests.java

```diff
@@ -138,4 +138,27 @@ public void testRpcValues () {
         assertNull(TestValues.NULL, msg.getRequestType());
         assertNull(TestValues.NULL, msg.getRequestSubType());
     }
+
```

Comment:
The javadoc for the `OnSystemRequest` still has max length defined. Can you remove that?

Reply:
@bilal-alsharifi Hi! Thanks for your feedback, I've provided changes. Please, check the PR